_data/people/matkinso.yml

```
@@ -41,3 +41,24 @@ presentations:
     meetingurl: https://exatrkx.github.io
     location: Virtual
     focus-area: ia
+  - title: "Graph Neural Networks Architectures"
+    date: 2020-10-21
+    url: https://indico.cern.ch/event/955026/contributions/4012924/attachments/2127589/3582355/IRIS-Topical_GNN_Tracking.pdf
```

```suggestion
    url: https://indico.cern.ch/event/955026/contributions/4012924/
```

@markusatkinson If you truncate the URLs to the contribution level then if anything ever happens to the PDF so that it changes the URL won't break.